library/Data/NBA/Stats.hs

```diff
@@ -65,6 +66,19 @@ getSplitRows ::
     -> IO (Either StatsError [a]) -- ^ The return value: an IO action resulting in an error or split rows.
 getSplitRows path splitName params = Except.runExceptT $ getSplitRowsGeneric path splitName params
 
+splitRows ::
+    (Aeson.FromJSON a)
+    => ResponseBody -- ^ The JSON string containing the splits
```

Comment:
I think the data should be the final parameter, because it will often be used as part of some sequence of effects such as `get url >>= splitRows splitName`.